requirements.txt

```diff
@@ -1,3 +1,9 @@
+six
+modin
```

Comment:
hmmm, i don't know if it's good idea to bring modin at this early stage, we still have work to do

Reply:
agreed - we can remove it. It is just some experiment I did sometime back on Modin.